de/ospo.md

```diff
@@ -28,3 +28,24 @@ Das OSPO bietet Beratung bei neuen Softwarelösungen (Beschaffungen, Ablöse, kl
 * Mitarbeit bei Fehlanzeigen, wenn es keine praktikable Open Source Lösung gibt.
 
 Das OSPO bietet nur beratende Mithilfe, keine Ausschreibungsdurchführung oder Make-Buy-Use-Compose Analysen (MBUC) an.
+
+
+## Team
+
+Neben zahlreichen Entwickler\*innen der städtischen Open Source Community, die Ideen, Leidenschaft und vor Public Code einbringen, kümmern sich um Open Source:
+
+![Dirk Gernhardt](/people/dirk_gernhardt__150.jpg#right)
```

Comment:
```suggestion
<img src="/people/dirk_gernhardt__150.jpg" class="right" alt="Dirk Gernhardt"></img>
```

I would prefer to use HTML where markdown isn't sufficient.

Reply:
but the internetz says: [this is not even a hack, but a very elegant method](https://stackoverflow.com/questions/22627458/how-to-add-class-in-image-markdown-in-ghost/22683624#22683624)